src/main/resources/logback.xml

```diff
@@ -5,9 +5,18 @@
       <pattern>%d{HH:mm:ss.SSS} %magenta([%thread]) %highlight(%-5level) %yellow(%logger{36}) - %msg%throwable%n</pattern>
     </encoder>
   </appender>
+  <appender name="FILE" class="ch.qos.logback.core.FileAppender">
```

Comment:
Logs to a file in a pod?
If this is needed for testing, move it to the test folder.
If not, please remove. Production logs of a pod should go always to stdout, so they do not produce excessive storage problems in the pod.

Reply:
It will be in the report